src/avatar/Avatar.js

```diff
@@ -124,9 +123,7 @@ const Avatar = props => {
       );
     } else if (title) {
       return (
-        <Text style={[styles.title, titleStyle && titleStyle]}>
```

Comment:
Strange, I've didn't touch these lines manually. Some crazy git hook? Will look into it.